tests/integration/kubernetes/kata-deploy-ensure-runtimec-classes-created.bats

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../common.bash"
+load "${BATS_TEST_DIRNAME}/tests_common.sh"
+
+setup() {
+	expected_runtime_classes=2
```

Comment:
Just to double check - the reason we have two expected runtime classes are the one we are testing and `kata` which defaults to map to it?

Reply:
Exaclty!  Do you want me to add a comment making it explicit? (I do think it's a good idea)

Reply:
I did.

Reply:
Thanks - that might have been what I was hinting at ;)